packages/app/src/cli/models/extensions/extension-specifications/checkout_ui_extension.ts

```diff
@@ -13,6 +13,7 @@ const CheckoutSchema = BaseExtensionSchema.extend({
 const spec = createExtensionSpec({
   identifier: 'checkout_ui_extension',
   externalIdentifier: 'checkout_ui',
+  externalName: 'Checkout UI',
   surface: 'checkout',
   dependency,
   partnersWebId: 'checkout_ui_extension',
```

Comment:
I find out nomenclature a little inconsistent around the ID.  We use ID and Indentifier:

* identifier
* externalIdentifier
*partnersWebId: string

Not a biggie, but could we be consistent?